tests/contrib/logging/test_tracer_logging.py

```diff
@@ -202,9 +204,9 @@ def test_logs_with_basicConfig(run_python_code_in_subprocess, ddtrace_run_python
 
         code = """
 import logging
-import ddtrace
```

Comment:
@P403n1x87 by removing this, does this interfere with line 208/210 below?
```python
ddtrace_logger = logging.getLogger('ddtrace')
```

Reply:
Yup. I'll fix this when #5103 is merged.